src/router/index.tsx

```diff
@@ -15,6 +16,9 @@ export default function Routes() {
         <Route exact path="/">
           <Home />
         </Route>
+        <Route exact path="/about-us">
```

Comment:
Please remove `exact` property from every path except the main one. (`/`)

Reply:
this file will be deleted after changing to Nextjs.